.github/workflows/announce-calls.yml

```diff
@@ -0,0 +1,23 @@
+name: Announce calls
+on:
+  schedule:
+    # 8th to the 14th of each month at 9am.
+    - cron: '0 9 8-14 * *'
+    # Thusdays at 9am.
+    - cron: '0 9 * * 4'
+
+jobs:
+  announce:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure Git.
+        run: |
+          git config --global user.name 'farmOS'
+          git config --global user.email 'noreply@farmOS.org'
+      - name: Monthly community call
+        if: github.event.schedule == '0 9 8-14 * *'
```

Comment:
Too bad I don't see a way to have "named schedule" events or something so this wouldn't be fragile to changes above...

Reply:
Yea agreed. Copied this pattern directly from the official docs here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule

Reply:
I suppose we could remove these `if:`s and just add `test $(date +\%u) -eq [day]` to both?

Reply:
eg:

```
      - name: Monthly community call
        run: test $(date +\%u) -eq 3 && git commit --allow-empty -m 'The #farmOS monthly community call is today at 2pm Eastern. All are welcome! https://farmos.org/community/monthly-call/' && git push origin main
      - name: Weekly dev call
        run: test $(date +\%u) -eq 4 && git commit --allow-empty -m 'The #farmOS weekly dev call is today at 12pm Eastern. All are welcome! https://meet.jit.si/farmos-dev' && git push origin main
```

That should work, right? They will each only run on Wed/Thu, respectively. Combined w/ the schedule cron rules I think that would have the desired effect.

Reply:
Force pushed. Thoughts @symbioquine?

Reply:
That's a cool approach! It might need a comment to remind us in future how that works...

Reply:
It might also be a good idea to explicitly set the timezone for the date command to match the UTC for the cron syntax.

Reply:
It could also just be two separate actions... :-)

Reply:
Yea thought about separating into two actions too... we would still need the `test $(date +\%u) -eq 3` for the monthly call anyway. So overall it would be more duplication IMO. This feels simple enough. If we ever need to make this more complex we can reconsider splitting.

I'll add comments and explicit timezone @symbioquine ...

Reply:
Decided to restructure into a single `step` with a `case`:

```
      # We use `test $(date +\%u)` below in combination with the schedule cron
      # rules above to ensure these only run on the days they are meant to. 
      - name: Announce calls
        run: |
          TZ=UTC
          DAY=$(date +\%u)
          case "${DAY}" in
              3)    git commit --allow-empty -m 'The #farmOS monthly community call is today at 2pm Eastern. All are welcome! https://farmos.org/community/monthly-call/' ;;
              4)    git commit --allow-empty -m 'The #farmOS weekly dev call is today at 12pm Eastern. All are welcome! https://meet.jit.si/farmos-dev' ;;
          esac
          git push origin main
```

What do you think?